assets/js/blocks/active-filters/types.ts

```diff
@@ -4,3 +4,14 @@ export interface Attributes {
 	displayStyle: string;
 	className?: string;
 }
+
+export interface ActiveFiltersBlockProps {
+	/**
+	 * The attributes for this block.
+	 */
+	attributes: Attributes;
+	/**
+	 * Whether it's in the editor or frontend display.
+	 */
+	isEditor: boolean;
+}
```

Comment:
I think it's better if we change
```ts
export interface Attributes {
	heading: string;
	headingLevel: number;
	displayStyle: string;
	className?: string;
}

export interface ActiveFiltersBlockProps {
	/**
	 * The attributes for this block.
	 */
	attributes: Attributes;
	/**
	 * Whether it's in the editor or frontend display.
	 */
	isEditor: boolean;
}
```
to
```ts
export interface ActiveFiltersBlockProps {
	heading: string;
	headingLevel: number;
	displayStyle: string;
	className?: string;
	isEditor?: boolean;
}
```
That way, it's consistent with the other components. The definition `isEditor?: boolean;` means that this value is optional.

Reply:
@nielslange 
I tried handling this before also but this gave a lot of errors to me as some files like `index.tsx` is calling for attributes property separately and if I gave the properties separately then also a lot of issues arise like `heading` being a duplicate since we already have a variable called `heading` and some more things.
For now, I've kept `Attributes` as a separate interface also. If you still want it to be changed, I will try my best.